README.md

```diff
@@ -6,6 +6,7 @@
 [![Open Source Love](https://badges.frapsoft.com/os/v3/open-source.svg?v=103)](https://github.com/anitab-org/open-source-programs-backend/)
 [![Maintenance](https://img.shields.io/badge/Maintained%3F-yes-green.svg)](https://github.com/anitab-org/open-source-programs-backend)
 ![PRs Welcome](https://img.shields.io/badge/PRs-welcome-brightgreen.svg?style=flat-square)
+[![codecov](https://codecov.io/gh/anitab-org/open-source-programs-backend/branch/develop/graph/badge.svg)](https://codecov.io/gh/anitab-org/open-source-programs-backend)
```

Comment:
@himanshu007-creator as the repo name has been changed to anitab-forms-backend please make the necessary changes.

Reply:
Yes even I thought that @codesankalp